pwa-devdocs/scripts/markdown-linter/index.js

```diff
@@ -0,0 +1,24 @@
+const linter = require('./linter');
+const fs = require('fs');
+const report = require('vfile-reporter');
+
+module.exports = {
+    linter: filepath => {
+        linter(filepath)
```

Comment:
Does this function not return the promise?

Reply:
`markdown-linter::linter` appears to return `void`, trapping the promise and breaking the chain returned by `linter`. `fixer` below doesn't trap, but I'm not sure why.

Without a docblock this API would be hard to understand; is there a best practice we should apply here?